src/librustdoc/html/static/js/search.js

```diff
@@ -2314,8 +2441,15 @@ function initSearch(rawSearchIndex) {
         } else {
             output = [];
         }
+        const where_clause = [];
+        const l = functionSearchType.length;
```

Comment:
Please rename it (took me a few seconds when reading the line below than `i < l` was not `i < 1`, damn font, not the main reason though, but at least my brain will stopped to be confused by this haha).